src/projects/projects-page.html

```diff
@@ -221,8 +221,8 @@
       _tapHideProjectList: function() {
         this.showProjectsList = !this.projectSelected;
       },
-      _hideProjectList: function(pullRequestSelected) {
-        this.showProjectsList = !pullRequestSelected;
+      _showProjectList: function(subpage) {
+        this.showProjectsList = (!subpage.subpage || subpage.subpage === 'activity');
```

Comment:
I think this does not solve the essence of the issue. Now, when you click `Activity`, the lists expand, which is something i would not expect.